internal/manifests/monolithic/configmap.go

```diff
@@ -193,23 +193,28 @@ func buildTempoConfig(opts Options) ([]byte, error) {
 		case v1alpha1.MonolithicTracesStorageBackendS3:
 			config.Storage.Trace.Backend = "s3"
 			config.Storage.Trace.S3 = &tempoS3Config{}
-			config.Storage.Trace.S3.Endpoint = opts.StorageParams.S3.Endpoint
-			config.Storage.Trace.S3.Insecure = opts.StorageParams.S3.Insecure
-			config.Storage.Trace.S3.Bucket = opts.StorageParams.S3.Bucket
-			if tempo.Spec.Storage.Traces.S3 != nil && tempo.Spec.Storage.Traces.S3.TLS != nil && tempo.Spec.Storage.Traces.S3.TLS.Enabled {
-				if tempo.Spec.Storage.Traces.S3.TLS.CA != "" {
-					config.Storage.Trace.S3.TLSCAPath = path.Join(manifestutils.StorageTLSCADir, opts.StorageParams.S3.TLS.CAFilename)
+			if opts.StorageParams.S3.LongLived != nil {
```

Comment:
Just to understand: can a S3 bucket be long-lived and short-lived at the same time? If not, should we raise and error if S3 Long and Short lived are configured at the same time?

Reply:
great catch, we should check for this